src/pages/app/laboratory/BillClient/query.ts

```diff
@@ -0,0 +1,22 @@
+const LabClientQuery = (facilityId?: string, val?: string) => {
```

Comment:
```suggestion
const LabClientQuery = (facilityId?: string, val?: string) => {
```


Can you make this a camel case name